src/style.css

```diff
@@ -0,0 +1,54 @@
+body {
+  margin: 0;
+}
+
+.nav-link {
+  position: relative;
+  font-family: "Roboto", sans-serif;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+  margin-left: 20px;
+  line-height: 60px;
+  color: #000;
+}
+
+.is-active {
```

Comment:
if we add this class to another element of the list?

![image](https://user-images.githubusercontent.com/44294945/64180224-17595b80-ce6d-11e9-95d2-c2c8cdb3b827.png)